src/Containers/DropArea/DropArea.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useMemo } from 'react';
+import styled, { useTheme } from 'styled-components';
+import { MainInterface, Main } from '@Utils/BaseStyles';
+import Dropzone, {
+    useDropzone,
+    DropEvent,
+    FileRejection,
+} from 'react-dropzone';
+
+export interface IDropAreaProps
+    extends MainInterface,
+        React.HTMLAttributes<HTMLDivElement> {
+    message?: string;
+    onDragEnter?: React.DragEventHandler<HTMLElement> | undefined;
+    onDragLeave?: React.DragEventHandler<HTMLElement> | undefined;
+    onDropHandler?:
+        | (<T extends File>(
+              acceptedFiles: T[],
+              fileRejections: FileRejection[],
+              event: DropEvent,
+          ) => void)
+        | undefined;
+    isDragEnter?: boolean;
+}
+
+export const DropArea: React.FC<IDropAreaProps> = ({
```

Comment:
Should be in styled component

Reply:
no, I need to pass isDragEnter as a prop